scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html

```diff
@@ -391,23 +391,27 @@ <h5>
       start_coord_field.disabled = true;
       end_coord_field.disabled = true;
     }
-    start_coord_field.value={{filters_form.position.data|safe}};
-    end_coord_field.value={{filters_form.end.data|safe}};
+    if("{{filters_form.position.data|safe}}" !== "None") {
+      start_coord_field.value ={{filters_form.position.data|safe}};
+    }
+    if("{{filters_form.end.data|safe}}" !== "None") {
+      end_coord_field.value ={{filters_form.end.data|safe}};
+    }
   }
 
   // Set cytobands select options according to selected chrom
   function setCytobands(){
     clearSelect(cytoStart)
     clearSelect(cytoEnd)
     // set cytoband selects options only if a chromosome is selected
-    selectedChrom = chromSelect.value;
+    let selectedChrom = chromSelect.value;
     let build = document.forms["filters_form"].elements["build"].value;
-    if (selectedChrom !== 'undefined' && selectedChrom !== "All" ){
+    if (selectedChrom !== undefined && selectedChrom !== "" && selectedChrom !== "All"){
```

Comment:
Directly comparing to `undefined` is the actual fix, the "None" issue was invisible before.